docs/changelog/working.rst

```diff
@@ -0,0 +1,84 @@
+SIMP Community Edition (CE) 6.3.0-Beta
```

Comment:
I would like to see this file called 'lastest.rst' to correspond to the 'latest' version selector as shown on read the docs